fullcalendar-app/src/components/CalendarHeader.vue

```diff
@@ -16,15 +16,19 @@
 
     <button class="legend-toggle-button btn btn-dark" data-mdb-ripple-init @click="toggleLegend">Legend +</button>
     <div :class="['checkbox-container', { active: isLegendOpen }]">
-      <label class="custom-checkbox" v-for="category in categories" :key="category.name">
+      <label
+        v-for="category in categories"
+        :key="category.name"
+        :style="{'border-color': category.color, 'background-color': 'color-mix(in srgb, ' + category.color +', transparent 80%)'}"
```

Comment:
would be good transparent have in css, not sure it is possible in current mix() case

Reply:
@podarok pushed changes in new commit. `color-mix` is pretty widely supported, but `rgba` is better. https://caniuse.com/?search=color-mix